.github/workflows/contrib-tests.yml

```diff
@@ -176,6 +176,45 @@ jobs:
           file: ./coverage.xml
           flags: unittests
 
+  WebSurfer:
```

Comment:
we also need a block in contrib-openai.yml

Reply:
I think I added what you are referring to (re: Docker). If not, please let me know specifically which block is missing.

Reply:
I mean we need to add a section in contrib-openai.yml to enable the openai test for test_web_surfer.py.

Reply:
Got it! Will do that shortly

Reply:
@sonichi  Done. But hard for me to independently verify. (The OpenAI tests run fine in my environment, but I don't have a way of testing the yml). 

Also, it will skip tests that involve Bing. If we can add a secrets.BING_API_KEY, then these tests will become enabled. I can provide that key.